L/libjulia/common.jl

```diff
@@ -330,13 +336,16 @@ function build_julia(ARGS, version::VersionNumber)
         push!(dependencies, BuildDependency(get_addable_spec("LLVM_full_jll", v"12.0.0+0")))
 
         # starting with Julia 1.7, we need LLVMLibUnwind_jll
-        push!(dependencies, BuildDependency(get_addable_spec("LLVMLibUnwind_jll", v"11.0.1+1")))
+        # this seems to replace libunwind.h and the build then fails on Linux?
+        #push!(dependencies, BuildDependency(get_addable_spec("LLVMLibUnwind_jll", v"11.0.1+1")))
     else
```

Comment:
OK, all 53 checks have passed, but I'm still nervous about having to remove LLVMLibUnwind_jll. Building with this enabled failed on Linux on my machine, would removing it cause problems?